pkgs/top-level/all-packages.nix

```diff
@@ -7818,7 +7818,9 @@ with pkgs;
 
   fcitx5 = libsForQt5.callPackage ../tools/inputmethods/fcitx5 { };
 
-  fcitx5-with-addons = libsForQt5.callPackage ../tools/inputmethods/fcitx5/with-addons.nix { };
+  fcitx5-with-addons = libsForQt5.callPackage ../tools/inputmethods/fcitx5/with-addons.nix {
+    makeWrapper = makeBinaryWrapper;
```

Comment:
Anything wrong with the default shell wrapper?

Reply:
It works but the I thought the C wrapper is better?

Reply:
Then just use `makeBinaryWrapper` directly, no need to override it here.